src/Apps/Artwork/Components/PricingContext.tsx

```diff
@@ -43,11 +43,23 @@ function PricingContext({ artwork }: PricingContextProps) {
             const artworkFallsInThisBin =
               artwork.priceCents.min >= bin.minPriceCents &&
               artwork.priceCents.min < bin.maxPriceCents
+            let labelDescription
+            let binValue = bin.numArtworks
+
+            if (bin.numArtworks > 1) {
+              labelDescription = bin.numArtworks + " works"
+            } else if (artworkFallsInThisBin && bin.numArtworks === 0) {
+              labelDescription = "1 work"
```

Comment:
Mind adding some tests for these new cases?

Also, I think we can make this a little simpler:
```js
const binValue = artworkFallsInThisBin && bin.numArtworks === 0 ? 1 : bin.numArtworks
const labelSuffix = bin.numArtworks > 1 ? " works" : " work"

...

description: binValue + labelSuffix
```

Reply:
@sweir27 I probably lack context, but shouldn't that be this?

```js
const labelSuffix = bin.numArtworks === 1 ? " work" : " works"
```

Reply:
@alloy that makes sense, although it wouldn't make a difference in this case since in the 0 case, the bar will have 0px height and therefore there is nowhere for you to hover to see the '0 works' label.

Reply:
Gotcha, thanks for the context 👍 I’d probably still suggest that change, for in case the design changes in the future.